dist/js/smooth-scroll.js

```diff
@@ -231,6 +240,14 @@
 		if ( type === 'easeInQuint' ) pattern = time * time * time * time * time; // accelerating from zero velocity
 		if ( type === 'easeOutQuint' ) pattern = 1 + (--time) * time * time * time * time; // decelerating to zero velocity
 		if ( type === 'easeInOutQuint' ) pattern = time < 0.5 ? 16 * time * time * time * time * time : 1 + 16 * (--time) * time * time * time * time; // acceleration until halfway, then deceleration
+
+		// Custom Easing Patterns
+		if ( settings.easingPatterns[type] ) {
+			pattern = settings.easingPatterns[type]( time );
+		}
+
+		console.log(pattern || time);
```

Comment:
Would it be possible to remove this `console.log`? It is littering the console :)